src/main/java/com/griefcraft/lwc/LWC.java

```diff
@@ -946,24 +950,49 @@ public void sendLocale(CommandSender sender, String key, Object... args) {
 
             // did they cancel it?
             if (evt.isCancelled()) {
-                return;
+                return null;
             }
         }
 
         if (message == null) {
             sender.sendMessage(Colors.Dark_Red + "LWC: " + Colors.White + "Undefined locale: \"" + Colors.Dark_Gray + key
                     + Colors.White + "\"");
-            return;
+            return null;
         }
 
         if (message.length > 0 && message[0].equalsIgnoreCase("null")) {
+            return null;
+        }
+
+        return message;
+    }
+
+    /**
+     * Send a locale to a player or console
+     *
+     * @param sender
+     * @param key
+     * @param args
+     */
+    public void sendLocale(CommandSender sender, String key, Object... args) {
```

Comment:
As mentioned in my previous comment, you can see there is a problem in this method, because it is going to throw an LWCSendLocaleEvent twice, and may also fail to send the message afterward.